build.js

```diff
@@ -30,6 +30,9 @@ snx.networks
 
 // additional
 redirects.push('/ProtocolDAO https://etherscan.io/address/0xEb3107117FEAd7de89Cd14D463D340A2E6917769 302');
+redirects.push('/GrantsDAOV2 https://etherscan.io/address/0x4cf117aaa757dad4a084025c3f23f1b67a037049 302');
+redirects.push('/grantsDAO https://etherscan.io/address/0xeb9a82736cc030fC4A4CD4b53e9B2c67e153208d 302');
```

Comment:
You don't want to use capitalized case like everything else?

Reply:
amended